sqlgen/reflect.go

```diff
@@ -815,3 +815,42 @@ func (t *Table) extractRow(row interface{}) Filter {
 
 	return f
 }
+
+func MakeCanonical(v interface{}) interface{} {
+	switch v := v.(type) {
+	case bool:
+		return bool(v)
+	case int8:
+		return int64(v)
+	case int16:
+		return int64(v)
+	case int32:
+		return int64(v)
+	case int64:
+		return int64(v)
+	case int:
+		return int64(v)
+	case uint8:
+		return uint64(v)
+	case uint16:
+		return uint64(v)
+	case uint32:
+		return uint64(v)
+	case uint64:
+		return uint64(v)
+	case uint:
+		return uint64(v)
+	case string:
+		return string(v)
+	case []byte:
+		return []byte(v)
+	case float32:
+		return float64(v)
+	case float64:
+		return float64(v)
+	case time.Time:
+		return time.Time(v)
```

Comment:
There are some types here that we're doing nothing with. Do we need to handle them?

Reply:
this handles all driver.Value.

Reply:
This handles more than `driver.Value` - for example, `uint32` is not a `driver.Value`, only `int64` is

Reply:
opened a separate PR https://github.com/samsarahq/thunder/pull/170